src/catalogue/aws/build.gradle

```diff
@@ -69,10 +69,6 @@ dockerBuildNative {
     images = ["iad.ocir.io/cloudnative-devrel/micronaut-showcase/mushop/${project.parent.name}-${project.name}-native:$project.version"]
 }
 
-graalvmNative {
```

Comment:
this can be removed as `db/migration` is used as the default (see https://micronaut-projects.github.io/micronaut-flyway/latest/guide/#graalvm)